acis_thermal_check/regression_testing.py

```diff
@@ -251,23 +233,26 @@ def compare_results(self, load_week, out_dir):
         new_results = pickle.load(open(new_answer_file, "rb"))
         old_answer_file = os.path.join(test_data_dir, self.name, load_week,
                                        "validation_data.pkl")
-        old_results = pickle.load(open(old_answer_file, "rb"))
+        kwargs = {}
+        if six.PY3:
+            kwargs["encoding"] = "bytes"
+        old_results = pickle.load(open(old_answer_file, "rb"), **kwargs)
```

Comment:
Getting all this unicode stuff right is a dark art...  I think you should be encoding as `ascii` for PY3 instead of `bytes`.  Basically as soon as you find yourself writing things like `old_results[b"pred"]` then something has gone wrong.  You should be able to do everything in Py2/Py3 compatible code using "natural" strings, i.e. `str` in both cases.

See:
https://github.com/sot/kadi/blob/b85bbc47fb9458357d2206bbdc883513f8510003/kadi/cmds/cmds.py#L54